src/components/ErrorPage/ErrorPage.jsx

```diff
@@ -0,0 +1,17 @@
+import './ErrorPage.css';
+
+const ErrorPage = () => {
+  const errorImage = require('../../assets/MindFlayerFront.png');
+
+  return (
+    <div className="ErrorPage">
+      <img src={errorImage} className='error-image' />
+      <div className="error-message">
+        {/* When we have error messages returned from BE, this copy will be replaced */}
+        Uh oh! You seem to have landed in an unknown realm. <br /><br /> Please Plane Shift your way back Home and try again.
```

Comment:
This is the first time I've seen that break tag, cool to know that it exists! I'll definitely try using it in the future.

Reply:
I don't think it's great form to use it a lot, but in this case I really wanted to break up the look of the message, so I snuck two of them in there to get the look I wanted. 😆  They'll probably come out when we start rendering the messages directly from BE instead of hard coding. But yeah, they're handy little buggers. 😉